src/document.js

```diff
@@ -20,8 +20,20 @@ const PostMessages = new PostMessageService({
 
 const showLoadingIndicator = () => document.getElementById('loadingContainer').classList.add('icon-loading')
 const hideLoadingIndicator = () => document.getElementById('loadingContainer').classList.remove('icon-loading')
+const showLoading = () => {
+	if (OC.appswebroots.richdocumentscode && Config.get('urlsrc').indexOf('proxy.php') >= 0) {
+		$.get(OC.appswebroots.richdocumentscode + '/proxy.php?status').done(function(val) {
```

Comment:
Please add some error handling here in case the request fails for some reason

Reply:
Ah actually the timeout handler should then just kick in but there seems to be an issue with the timeout postmessage not reaching the parent frame. I'll have a look at that. Let's get this in so far.

Reply:
I want to make change before get this in thanks